[Doc] Update Radio Button Website: Selectable Components (#3890)
carbon-design-system/carbon-website: src/pages/components/radio-button/usage.mdx

1. **Group label (optional):** Describes the group of options or provides
   guidance for making a selection.

> 2. **Radio button input:** A radio button indicating the appropriate state. By

Reviewer: ```suggestion
2. **Radio button input:** Indicates the state of a radio button. By
```

>    default no option will be preselected.

Reviewer: ```suggestion
   default, no option will be preselected.
```

### States

The radio button input allows for two states: **unselected** and **selected**.
The default view of a radio button is having no radio button preselected. Only
one radio button should be selected at a time. When a user chooses a new item,
the previous choice is automatically deselected. If the user has already

> selected an item but wanted to deselect it, consider adding alternatives such as

Reviewer: ```suggestion
selected an item but wants to deselect it, consider adding alternatives such as
```
